com.github.sdbg.debug.ui/plugin.xml

```diff
@@ -1077,6 +1079,15 @@
             <bundle name="com.github.sdbg.debug.core"/>
       </component>
    </extension>
+   
+   <!-- Used to launch chrome from another plugin like GWT -->
+   <extension
+         point="com.github.sdbg.debug.ui.launchChrome">
```

Comment:
I would change the extension point ID to com.github.sdbg.debug.ui.launchers.chrome to have a better correlation to the new package.